### PR TITLE
US002 view job spec

### PR DESCRIPTION
**Pull request by OdhranHague**
As a Kainos Employee

I want to find job specification information about each role
So that I know what is expected for each role

Acceptance Criteria:

- [x] Able to click on the job role in the table 

- [x] Can view job spec summary for each role

- [x] Can view and follow link to Job Spec for each role in SharePoint"

This is what the UI page currently looks like:
Screenshot 2023-12-18 at 11 07 22

After clicking on the name of the role you can view it in full:
Screenshot 2023-12-19 at 09 25 50

### CODE REVIEW

**Review by Joseph-Magee — APPROVED** [review]
looks good

File: validator/roleValidator.ts
Comment (jkelso662) [inline validator/roleValidator.ts:1]: ❤️

File: test/unit/service/roleServiceTests.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { expect } from "chai";
+const roleService = require("../../../service/roleService");
+
+const role = {
+    roleName : "role",
+    specification : "ijasndjsanknajn spec entered bla bla",
+    capabilityName : "engineering",
+    responsibilities : "be very responsible",
+    sharepointLink : "https://kainossoftwareltd.sharepoint.com/SitePages/Home.aspx",
+};
```
Comment (jkelso662) [inline test/unit/service/roleServiceTests.ts:12]: This wouldn't get into a real project repo. Needs to be at least sensible info!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Great! Really good PR just two wee things to fix - make sure you're we are treating this like a real Kainos project repo.

**Review by jkelso662 — APPROVED** [review]
👍

**Pull request merged**